src/pages/Login/Login.tsx

```diff
@@ -1,17 +1,64 @@
 import React, { FunctionComponent } from 'react';
 import styled from 'styled-components';
+import { TopNav } from 'components/TopNav/TopNav';
+import LogoSvg from 'assets/logo.svg';
 import { useMutation } from '@apollo/react-hooks';
 import { Firebase, useFirebase } from 'global/Firebase';
 import { SIGN_IN, SignInResponse } from 'global/graphqls/SignIn';
 import { PS_TOKEN_NAME } from 'global/ApolloClient/ApolloClient';
 import { Button } from 'components/Button';
-import { Input } from 'components/Input';
-import { Icon as IconComponent } from 'components/Icon';
-import EmailIconSvg from 'assets/email.svg';
-import { Checkbox } from 'components/Checkbox';
+import { EmailInput } from 'components/EmailInput/EmailInput';
 import { PasswordInput } from 'components/PasswordInput';
+import { Card } from 'components/Card/Card';
+import { Link } from 'react-router-dom';
 
-const RedButton = styled(Button)`
+const Caption = styled.span(props => props.theme.typography.caption);
+
+const PageWrapper = styled.div`
```

Comment:
This whole code looks copied and pasted from Register site. Maybe it's good to extract the repetitive part and do it once. In future we can change it only in once place.